src/test/java/io/jenkins/plugins/grading/PitScoreTest.java

```diff
@@ -38,17 +37,121 @@ void shouldCalculateRatioImpacts() {
+        PitConfiguration pitConfiguration = createConfiguration(0, 0, 0, 0);
+        PitBuildAction pitBuildAction = createAction(1, 20);
+
+        PitScore pits = new PitScore(pitConfiguration, pitBuildAction);
+
+        assertThat(pits.getUndetectedSize()).isEqualTo(20);
+    }
+
+    @Test
+    void shouldGetDetectedSize() {
+        PitConfiguration pitConfiguration = createConfiguration(0, 0, 0, 0);
+        PitBuildAction pitBuildAction = createAction(30, 20);
+
+        PitScore pits = new PitScore(pitConfiguration, pitBuildAction);
+
+        assertThat(pits.getDetectedSize()).isEqualTo(10);
+    }
+
+    @Test
+    void shouldGetRatio() {
+        PitConfiguration pitConfiguration = createConfiguration(0, 0, 0, 0);
+        PitBuildAction pitBuildAction = createAction(100, 50);
+
+        PitScore pits = new PitScore(pitConfiguration, pitBuildAction);
+
+        assertThat(pits.getMutationsSize()).isEqualTo(100);
+        assertThat(pits.getUndetectedSize()).isEqualTo(50);
+        assertThat(pits.getDetectedSize()).isEqualTo(50);
+        assertThat(pits.getRatio()).isEqualTo(50);
+    }
+
+    @Test
+    void shouldCreateSimplePitScores() {
+        PitConfiguration pitConfiguration = createConfiguration(45, -10, 10, 10);
+        PitBuildAction pitBuildAction = createAction(80, 20);
+        PitScore pits = new PitScore(pitConfiguration, pitBuildAction);
+
+        assertThat(pits.getId()).isEqualTo(pitBuildAction.getDisplayName());
+        assertThat(pits.getTotalImpact()).isEqualTo(650);
+        assertThat(pits.getMutationsSize()).isEqualTo(80);
+        assertThat(pits.getUndetectedSize()).isEqualTo(20);
+        assertThat(pits.getDetectedSize()).isEqualTo(60);
+        assertThat(pits.getRatio()).isEqualTo(25);
+
+        PitConfiguration pitConfiguration2 = createConfiguration(10, 0, 5, 0);
+        PitBuildAction pitBuildAction2 = createAction(50, 10);
+        PitScore pits2 = new PitScore(pitConfiguration2, pitBuildAction2);
+
+        assertThat(pits2.getId()).isEqualTo(pitBuildAction2.getDisplayName());
+        assertThat(pits2.getTotalImpact()).isEqualTo(200);
+        assertThat(pits2.getMutationsSize()).isEqualTo(50);
+        assertThat(pits2.getUndetectedSize()).isEqualTo(10);
+        assertThat(pits2.getDetectedSize()).isEqualTo(40);
+        assertThat(pits2.getRatio()).isEqualTo(20);
+
+        PitConfiguration pitConfiguration3 = createConfiguration(9000, 500, -500, 10);
+        PitBuildAction pitBuildAction3 = createAction(80, 20);
+        PitScore pits3 = new PitScore(pitConfiguration3, pitBuildAction3);
+
+        assertThat(pits3.getId()).isEqualTo(pitBuildAction3.getDisplayName());
+        assertThat(pits3.getTotalImpact()).isEqualTo(-19750);
+        assertThat(pits3.getMutationsSize()).isEqualTo(80);
+        assertThat(pits3.getUndetectedSize()).isEqualTo(20);
+        assertThat(pits3.getDetectedSize()).isEqualTo(60);
+        assertThat(pits3.getRatio()).isEqualTo(25);
```

Comment:
Auch hier ist mir der Zweck der verschiedenen Zahlenwerte nicht klar

Reply:
(Wenn Sie verschiedene Kombinationen testen wollen, ist evtl. ein parametrisierter Test sinnvoller.)..

Reply:
Wenn ich den parametrisierte Tests verwende, würde ich dann alle bestehenden Tests durch parametrisierte ersetzen?

Reply:
Dann ja, aber ich denke in der aktuellen Fassung ist ein normaler Test leichter nachvollziehbar.